src/@types/ui-types.ts

```diff
@@ -5,6 +5,31 @@ import type { PlayerPokemon } from "#field/player-pokemon";
 import type { PokemonMove } from "#field/pokemon-move";
 import type { PokemonHeldItemModifier } from "#modifier/modifier";
 
+export interface ShowTextOptions {
+  /**
+   * The delay in milliseconds before the dialogue is displayed.
+   * @defaultValue `20`
+   */
+  delay?: number;
+  /** A callback function to execute after the dialogue is displayed. */
```

Comment:
```suggestion
  /** A callback function to execute after the text is done displaying. */
```
is that correct? Is it called when the text has finished appearing one letter at a time?

Reply:
I assume so.